mp/src/game/client/game_controls/SpectatorGUI.cpp

```diff
@@ -559,30 +568,29 @@ void CSpectatorGUI::OnThink()
 			}
 		}
 
-		static Label* jinScoreValueLabel = NULL;
-		static Label* nsfScoreValueLabel = NULL;
-		if (!jinScoreValueLabel)
+#ifdef NEO
+		if (!m_scoreValueLabelJinrai)
 		{
-			jinScoreValueLabel = dynamic_cast<Label*>(FindChildByName("CTScoreValue"));
+			m_scoreValueLabelJinrai = static_cast<Label*>(FindChildByName(LABEL_JINRAI));
 		}
-		if (!nsfScoreValueLabel)
+		if (!m_scoreValueLabelNSF)
 		{
-			nsfScoreValueLabel = dynamic_cast<Label*>(FindChildByName("TERScoreValue"));
+			m_scoreValueLabelNSF = static_cast<Label*>(FindChildByName(LABEL_NSF));
 		}
-		Assert(jinScoreValueLabel);
-		Assert(nsfScoreValueLabel);
+		Assert(m_scoreValueLabelJinrai);
+		Assert(m_scoreValueLabelNSF);
```

Comment:
Would it be more efficient to move these Asserts into the if statements above?

Reply:
Yes, but they're asserts so I wouldn't worry too much about it.

Reply:
The assert macros will be defined as essentially no-ops for release builds, so there's no runtime cost outside debug

```cpp
#define  Assert(_exp)    ((void)0)
```
etc.